### ISSUE TITLE
Remove override of createJSModules from OAuthManagerPackage.java

### ISSUE THREAD

**Issue by ujeenator** [fullstackreact/react-native-oauth#151]
The current version of React Native 0.47 has removed the createJSModules method in ReactPackage. Therefore any overrides of this method will need to be removed from the library, or this will cause a compilation error.

https://github.com/facebook/react-native/commit/ce6fb337a146e6f261f2afb564aa19363774a7a8

<img width="755" alt="screen shot 2017-08-28 at 6 40 51 pm" src="https://user-images.githubusercontent.com/1893760/29781079-a8ef8858-8c20-11e7-97af-cfcf886f932a.png">

**Comment by freeworlder** [comment]
Yes, since RN 0.47 it's really necessary. I have to manually clean createJSModules every time.

**Comment by MBach** [comment]
Yes please! Make a new release without the override annotation.
